plugin/documents.py

```diff
@@ -596,7 +597,24 @@ def _do_code_actions(self) -> None:
             self.view, covering, diagnostics_by_config, self._on_code_actions, manual=False)
 
     def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
-        action_count = sum(map(len, responses.values()))
+
+        def is_quickfix(kind: str) -> bool:
+            if kind == CodeActionKind.QuickFix:
+                return True
+            if len(kind) > len(CodeActionKind.QuickFix):
+                return kind.startswith(CodeActionKind.QuickFix) and kind[len(CodeActionKind.QuickFix)] == '.'
+            return False
```

Comment:
No problem, feel free to create a PR, then I can close this one.

I just wanted to get this started because it's a simple change and some of the code actions currently shown in the view are slightly annoying.

I think eventually we should also add filtering for the `only_kinds` case (as mentioned in the opening post).

Reply:
https://github.com/sublimelsp/LSP/pull/2081